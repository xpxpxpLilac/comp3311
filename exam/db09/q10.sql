drop view if exists allbrew;
create view allbrew
as
select distinct bs.id
from beers b join brewers bs on b.brewer = bs.id
;
drop view if exists nobrew;
create view nobrew
as
select bs.id as id, name
from brewers bs
where bs.id not in (select * from allbrew)
;
drop view if exists q10;
create view q10
as
select n.name as brewer,l.country as country
from nobrew n join brewers bs on bs.id = n.id
 		join locations l on bs.location = l.id 
;
