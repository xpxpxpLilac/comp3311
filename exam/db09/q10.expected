Klosterbrauhaus Ursberg|Germany
Liefmans Breweries|Belgium
Thunder Mountain Brewery|United States
Titanic Brewing|United States
